Resources/public/js/components/bulk-price/main.js

```diff
@@ -132,36 +135,84 @@ define(['text!suluproduct/components/bulk-price/bulk-price.html'], function(Bulk
                 }
             }.bind(this));
 
+            // special prices
+            var specialPrice = {};
+            specialPrice.currency = {};
+            specialPrice.start = $('#husky-input-js-dateFrom' + this.options.currency.code).val();
+            specialPrice.end = $('#husky-input-js-dateTo' + this.options.currency.code).val();
+            specialPrice.price = $('#js-input' + this.options.currency.code).val();
+            specialPrice.currency = this.options.currency;
+
+            this.sandbox.dom.data(this.$el, 'itemsSpecialPrice', specialPrice);
             this.sandbox.dom.data(this.$el, 'items', priceItems);
             this.sandbox.emit('sulu.products.bulk-price.changed');
+        },
+
+        initDateComponents = function (dateIds) {
+            this.sandbox.start([
+                {
+                    name: 'input@husky',
+                    options: {
+                        el: '#' + dateIds.from,
+                        instanceName: dateIds.from,
+                        skin: 'date'
+                    }
+                }
+            ]);
+            this.sandbox.start([
+                {
+                    name: 'input@husky',
+                    options: {
+                        el: '#' + dateIds.to,
+                        instanceName: dateIds.to,
+                        skin: 'date'
+                    }
+                }
+            ]);
         };
 
     return {
 
         initialize: function() {
-            var prices = [], salesPrice;
+            var prices = [],
+            salesPrice,
+            specialPrice = {},
+            dateIds = {};
```

Comment:
```dateIds``` is distracting. as this is an object it is not a list of ids as the name would suggest